tests/Telemetry/ChatTelemetryTests.cs

```diff
@@ -0,0 +1,314 @@
+﻿using NUnit.Framework;
+using OpenAI.Chat;
+using OpenAI.Telemetry;
+using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Linq;
+using System.Net.Sockets;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using static OpenAI.Tests.Telemetry.TestMeterListener;
+using static OpenAI.Tests.Telemetry.TestActivityListener;
+
+namespace OpenAI.Tests.Telemetry;
+
+[TestFixture]
+[NonParallelizable]
+public class ChatTelemetryTests
```

Comment:
If I'm reading these correctly, these are basically mock tests, right? If so, could you add the attribute `[Category("Smoke")]` to this `ChatTelemetryTests class?